### PR TITLE
Script creates multiple VD and max config, also runs CC operations_v3

### PR DESCRIPTION
**Pull request by vrbagalkote**
This script Creates 16VD for each raid array, and 64VD for raid0
and runs CC operations

Signed-off-by: Venkat R B <vrbagal1@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vrbagalkote** [comment]
Attached is the job.log file.
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/648898/job.txt)

File: io/disk/Avago_storage_adapter/avago9361_vd.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                          'r00': [4, 'Multiple2'], 'r10': [4, 'Multiple2'],
+                          'r50': [6, 'Multiple3'], 'r60': [6, 'Multiple3']}
+        self.value = self.dict_raid[self.raid_level]
+
+        if len(self.disk) < self.value[0]:
+            self.skip("Please give enough number of drives to create %s"
+                      % self.raid_level)
+        if self.value[1] is not None:
+            multiple = int(self.value[1].split("Multiple")[-1])
+            print multiple
```
Comment (narasimhan-v) [inline io/disk/Avago_storage_adapter/avago9361_vd.py]: Think you forgot to remove it ?

**Comment by narasimhan-v** [comment]
Except for the one comment, looks good to me.

**Comment by vrbagalkote** [comment]
Removed the print statement.

**Comment by PraveenPenguin** [comment]
@vrbagalkote @narasimhan-v thanks

**Pull request merged**